.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
+env:
+  - CC_TEST_REPORTER_ID=f5b40344b76ccf66fbab5aa40bd5db671464c74f255cc3d4063f8bcd7c0812cf
```

Comment:
Can this ID be used as an authentication key? What does this ID represent?

Reply:
The ID is a repository identifier used by the CodeClimate test reporter to associate the reported coverage with the correct repository on CodeClimate.
As per, https://docs.codeclimate.com/docs/finding-your-test-coverage-token#section-should-i-keep-my-test-reporter-id-secret, the ID can't be used to access any data.

Reply:
alright. sound good to me